src/main/kotlin/com/creeperface/nukkitx/chestshop/ChestShop.kt

```diff
@@ -102,7 +98,77 @@ class ChestShop : PluginBase(), Listener {
 
         val line1 = lines[0].toLowerCase().trim()
 
-        if ((line1 == "[shop]" || line1 == "/shop")) {
+        if ((line1 == "sell" || line1 == "[sell]" || line1 == "/sell")) {
```

Comment:
Thought, could this be `line1.contains("sell")` does it need to be so strict?
You need to run the create command first, so it would be really hard to accidently trigger the command.